usbsdmux/i2c_gpio.py

```diff
@@ -40,6 +40,9 @@ def __init__(self, sg):
         """
         self._usb = Usb2642(sg)
 
+    def get_usb(self):
+        return self._usb
+
```

Comment:
I think we should instead turn around how `USB2642` and `I2cGpio` are set up in `usbsdmux.py`.
We should initialize the `USB2642` in `usbsdmux.py` directly and give `I2cGpio::__init__()` a reference to it on its initialization.

This way we have both the `USB2642` and `I2cGpio` in `usbsdmux.py` without this kind of trickery.

Reply:
That would be my preferred approach as well, but I didn't want to mix that refactoring into this PR. In a follup-up to this one, the benefit would be much clearer.